share/doc/wake/quickref.md

```diff
@@ -249,16 +249,26 @@ Runners are created using `makeRunner`. `makeRunner` is defined as follows:
 
 Environment packages can be added to a workspace to provide tools (using runners) for running the workflow. They are composed of Wake files that define and publish runners. Environment packages should be defined for each unique running environment to decouple Wake rules from how tools are installed in each environment. You can check out [environment-example-sifive](https://github.com/sifive/environment-example-sifive) as an example of an environment package that contains runners for Wake-based workflows. 
 
-#### sources and Path objects
+#### Sources and `Path` objects
 
-Sources are the set of files in git.
+Sources are the set of files in a Git repository. A `Path` in Wake is either a) a file in Git or b) a file produced from a build step. The `Path` type has no public constructor to help enforce this. To get a `Path` value, use the
 
-A Path is either a) a file in git or b) a file produced from a build step
+* `source: (file: String) => Result Path Error`
+* `sources: (dir: String) => (filterRegexp: RegExp) => Result (List Path) Error`
 
-To get a path, use the source function. For example, all the header files in this directory:
+functions. For example, to get all the header files in the current repository inside a Wake source file located at the workspace root:
 
+```wake
+sources @here `.*\.h`
 ```
```

Comment:
Rather than trying some circumlocution to fit `@here` in the initial explanation, why not make this initial example just be `"."` and have a later one describing how to use `@here`?  You may want a more descriptive example/situation than my quick sketch for the second part, though.
````suggestion
functions. For example, to get all the header files in the current repository:

```wake
sources "." `.*\.h`
```
````
`Note that the directory...`
````markdown
This allows you to easily reference a subset of the source files just by saving the Wake file in the enclosing directory:

```
sources @here `.*`
```
````

Reply:
That's a good suggestion to separate those out! [My attempt based on this feedback.](https://github.com/sifive/wake/pull/1009/commits/11fb18a193cbc4f9462252584d312a64fe1a84ff)